app/assets/javascripts/cardboard/new_tab_links.js

```diff
@@ -1,8 +1,8 @@
-$(document).on("page:update", function() {
```

Comment:
@leonelgalan This should go away when I merge master in, I think. I remember you reverting (correctly) the "links outside of Cardboard should open in new tabs" change because it broke the logout.